sdk/cognitivelanguage/azure-ai-language-conversations/samples/reslotion_and_pii/sample_conv_pii_transcript_input.py

```diff
@@ -0,0 +1,134 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+FILE: sample_conv_pii_transcript_input.py
+
+DESCRIPTION:
+    This sample demonstrates how to analyze a conversation for PII (personally identifiable information).
+
+    For more info about how to setup a CLU conversation project, see the README.
+
+USAGE:
+    python sample_conv_pii_transcript_input.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_CONVERSATIONS_ENDPOINT                       - endpoint for your CLU resource.
+    2) AZURE_CONVERSATIONS_KEY                            - API key for your CLU resource.
+"""
+
+from azure.ai.language import conversations
+
+
+def sample_conv_pii_transcript_input():
```

Comment:
Could we add async versions? :)

Reply:
i'm just waiting for final fixes to service apis in order to make sure no more changes in sync samples/tests